documentacion/pos-autoservicio/README.md

```diff
@@ -487,7 +487,7 @@ El objeto InitializationResponse retornará un objeto con los siguientes datos:
 
 ### Voucher
 
-Los voucher serán generados por el POS Autoservicio cuando el párametro `Enviar voucher` sea verdadero, el voucher puede ser retornado en la respuesta de las transacciones de venta, venta multicódigo, última venta, anulación y cierre.
+Los voucher serán generados por el POS Autoservicio cuando el párametro `Enviar voucher` sea verdadero, el voucher puede ser retornado en la respuesta de las transacciones de venta, <!-- venta multicódigo -->, última venta, anulación y cierre.
```

Comment:
Do these comments work on the pruduction site ?

Also, you have an extra comma

Reply:
Comments work in production 👍